src/dispatch/plugins/dispatch_slack/incident/interactive.py

```diff
@@ -1514,6 +1517,108 @@ def handle_assign_role_submission_event(
     )
 
 
+def handle_create_task_command(
+    ack: Ack,
+    body,
+    dict,
+    client: WebClient,
+    context: BoltContext,
+    db_session: Session,
+) -> None:
+    """Displays a modal for task creation."""
+    ack()
+    if context["subject"].type == CaseSubjects.case:
+        raise CommandError("Command is not currently available for cases.")
+
+    participants = participant_service.get_all_by_incident_id(
+        db_session=db_session, incident_id=context["subject"].id
+    )
+
+    incident = incident_service.get(db_session=db_session, incident_id=context["subject"].id)
+
+    contact_plugin = plugin_service.get_active_instance(
+        db_session=db_session, project_id=incident.project.id, plugin_type="contact"
+    )
+    if not contact_plugin:
+        raise CommandError(
+            "Contact plugin is not enabled. Unable to list participants.",
+        )
+
+    active_participants = [p for p in participants if p.active_roles]
+    participant_list = []
+
+    for participant in active_participants:
+        participant_email = participant.individual.email
+        participant_info = contact_plugin.instance.get(participant_email, db_session=db_session)
+        participant_name = participant_info.get("fullname", participant.individual.email)
+        participant_list.append({"text": participant_name, "value": participant_email})
+
+    blocks = [
+        static_select_block(
+            label="Participant",
+            block_id=CreateTaskBlockIds.participant_select,
+            placeholder="Select Participant",
+            options=participant_list,
+        ),
+        Input(
+            label="Task Description",
+            element=PlainTextInput(
+                placeholder="Task description", initial_value="description", multiline=True
+            ),
+            block_id=CreateTaskBlockIds.description,
+        ),
+    ]
+
+    modal = Modal(
+        title="Create Task",
+        blocks=blocks,
+        submit="Create",
+        close="Close",
+        callback_id=CreateTaskActionIds.submit,
+        private_metadata=context["subject"].json(),
+    ).build()
+    client.views_open(trigger_id=body["trigger_id"], view=modal)
```

Comment:
The first time this works fine but trying to issue the command again I get 
```
slack_sdk.errors.SlackApiError: The request to the Slack API failed. (url: https://slack.com/api/views.open)
The server responded with: {'ok': False, 'error': 'expired_trigger_id'}
File "/Users/dwhittaker/dev/dispatch/src/dispatch/plugins/dispatch_slack/incident/interactive.py", line 1580, in handle_create_task_command
    client.views_open(trigger_id=body["trigger_id"], view=modal)
    ```

Reply:
I'm not able to replicate this error.